src/openttd.cpp

```diff
@@ -625,6 +625,7 @@ int openttd_main(std::span<char * const> arguments)
 					SetDParamStr(0, _load_check_data.error_msg);
 					fmt::print(stderr, "{}\n", GetString(_load_check_data.error));
 				}
+				ret = 1;
 				return ret;
```

Comment:
```suggestion
				return 1;
```

And I won't mind if you fix that too in line 612. I have no clue why anyone would write it like that :P